internal/gatewayapi/testdata/securitypolicy-status-conditions.out.yaml

```diff
@@ -286,7 +286,7 @@ securityPolicies:
         namespace: envoy-gateway
       conditions:
       - lastTransitionTime: null
-        message: Unable to target HTTPRoute, another SecurityPolicy has already attached
+        message: Unable to target HTTPRoute httproute-1, another SecurityPolicy has already attached
```

Comment:
should we add namespace here?

Reply:
This makes sense. However, the info about the route's namespace is missing. See #3918